### PR TITLE
Supports different logging types in livevalidator

### PR DESCRIPTION
**Pull request by raych1**
Added loggingType to differentiate the log;
Added dedicated perf loggings.

### CODE REVIEW

File: lib/liveValidation/liveValidator.ts [hunk 1]
```diff
@@ -196,25 +214,43 @@ export class LiveValidator {
       }
     }
 
-    this.logging("Transforming all specs.");
+    const startGlobalTransform = Date.now();
+    this.logging("Apply global transforms for all specs");
     this.loader.transformLoadedSpecs();
```
Comment (PhoenixHe-msft) [inline lib/liveValidation/liveValidator.ts:219]: No need to perf this one. This one is super fast.

File: lib/liveValidation/operationValidator.ts [hunk 2]
```diff
@@ -75,11 +81,22 @@ export const validateSwaggerLiveRequest = async (
     }
     const startTimeToBuild = Date.now();
     validate = await loader.getRequestValidator(operation);
+    const elapsedTime = Date.now() - startTimeToBuild;
     if (logging) {
       logging(
-        `On-demand build request validator with DurationInMs:${Date.now() - startTimeToBuild}`,
+        `On-demand build request validator with DurationInMs:${elapsedTime}`,
```
Comment (PhoenixHe-msft) [inline lib/liveValidation/operationValidator.ts:87]: Is it possible to aggregate this metric in kusto?

**Review by PhoenixHe-msft — APPROVED** [review]

File: lib/liveValidation/operationValidator.ts (hunk 2 above)
Comment (raych1) [reply lib/liveValidation/operationValidator.ts:87]: Yes, we can.

File: lib/liveValidation/liveValidator.ts (hunk 1 above)
Comment (raych1) [reply lib/liveValidation/liveValidator.ts:218]: Removed it.

**Pull request merged**